samples/ChatSample/RedisUserTracker.cs

```diff
@@ -106,7 +106,7 @@ private static IConnectionMultiplexer ConnectToRedis(RedisOptions options, ILogg
             var loggerTextWriter = new LoggerTextWriter(logger);
             if (options.Factory != null)
             {
-                return options.Factory(loggerTextWriter);
+                return options.Factory(loggerTextWriter).Result;
```

Comment:
😅

Reply:
Gotta keep pushing `async` up the stack :)